### PR TITLE
Release v0.11.7 Part.1 (Bump Dependencies)

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 # crates.io
 futures         = { version = "0.3" }
-parity-util-mem = { version = "0.9", default-features = false, features = ["jemalloc-global"] }
+parity-util-mem = { version = "0.10", default-features = false, features = ["jemalloc-global"] }
```
Comment (AsceticBear) [inline Cargo.toml:21]: Do we need this explicit dependency? I did not find usage anywhere in this repo or darwinia-common.

**Review by AsceticBear — APPROVED** [review]
LGTM!

File: runtime/darwinia/src/pallets/fee_market.rs [hunk 2]
```diff
@@ -12,7 +12,6 @@ frame_support::parameter_types! {
 	pub const TreasuryPalletId: PalletId = PalletId(*b"da/trsry");
 	pub const FeeMarketLockId: LockIdentifier = *b"da/feelf";
 
-	pub const AssignedRelayersNumber: u64 = 3;
```
Comment (AsceticBear) [inline runtime/darwinia/src/pallets/fee_market.rs:15]: The newly added storage item `AssignedRelayersNumber` 's default value is 3. So nothing to do extra.

**Comment by HackFisher** [comment]
We may need to handle the removal migration of `ConfirmedMessagesThisBlock`?

**Comment by AsceticBear** [comment]
> We may need to handle the removal migration of `ConfirmedMessagesThisBlock`?

Good catch! we need it.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**